pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -124,14 +122,16 @@ def _save_ibmq_auth(qiskit_config: Optional[QiskitConfig]) -> None:
     token = None
     if qiskit_config is not None:
         token = qiskit_config.ibmq_api_token
-    if not IBMQ.active_account():
-        if IBMQ.stored_account():
-            IBMQ.load_account()
+    if token is None and os.getenv("PYTKET_REMOTE_QISKIT_TOKEN") is not None:
+        token = os.getenv("PYTKET_REMOTE_QISKIT_TOKEN")
+    try:
+        IBMProvider()
+    except:
+        if token is not None:
+            IBMProvider.save_account(token, overwrite=True)
+            IBMProvider()
         else:
-            if token is not None:
-                IBMQ.save_account(token)
-            else:
-                raise NoIBMQAccountError()
+            raise NoIBMQAccountError()
```

Comment:
Should this give more details? (That we have not been able to load a token for example?)

Reply:
The error message that will be given when this is raised will be the following 

```
No IBMQ credentials found on disk, store your account using qiskit, or using :py:meth:`pytket.extensions.qiskit.set_ibmq_config` first.
```